cmd/ui/src/views/Explore/ExploreSearch/ExploreSearch.tsx

```diff
@@ -96,25 +87,11 @@ const ExploreSearch = ({ onTabChange = () => {} }: ExploreSearchProps) => {
     };
 
     return (
-        <Box
-            sx={{
-                height: '100%',
-                display: 'flex',
-                flexDirection: 'column',
-                minHeight: 0,
-                gap: 1,
-            }}>
-            <Paper
-                sx={{
-                    height: '40px',
-                    display: 'flex',
-                    flexShrink: 0,
-                    gap: 1,
-                    backgroundColor: theme.palette.neutral.secondary,
-                    borderRadius: '8px',
-                    pointerEvents: 'auto',
-                }}
-                elevation={0}>
+        <div
+            className={cn('h-full h-min-0 w-[410px] flex gap-2 flex-col rounded-lg shadow-[1px solid white]', {
+                'w-[600px]': activeTab === 2 && showSearchWidget,
```

Comment:
I love to see these MUI styles dwindle!

Small thing for the active tab check -- instead of checking for 2 here (feels kinda magic number-y), could we use the tabNameMap defined above?

```suggestion
                'w-[600px]': activeTab === tabNameMap.cypher && showSearchWidget,
```